Libraries/QtVgCommon/vgGeoUtil.cxx

```diff
@@ -116,10 +116,10 @@ QString dmsString(double coord, const QString& dp, const QString& dn,
       resultTemplate = QStringLiteral("%1d%2\'%3\"%4");
       break;
     case vgGeodesy::FormatDmsLatin1:
-      resultTemplate = QStringLiteral("%1\u00b0%2\'%3\"%4");
+      resultTemplate = QStringLiteral(u"%1\u00b0%2\'%3\"%4");
```

Comment:
This change had no obvious effect on the (VS2017) build.

Reply:
> Visual Studio clobbers characters that are *outside the local code page*

(Emphasis added.) The degree symbol (°) is part of [Latin-1](https://en.wikipedia.org/wiki/ISO/IEC_8859-1) and probably happens to be in your local character set, and so does not get butchered.

However, it's safer to make all non-ASCII literals explicitly Unicode.